mobile/src/bundles/common/components/avatar-picker/avatar-picker.tsx

```diff
@@ -96,15 +101,37 @@ const AvatarPicker = <T extends FieldValues>({
 
     return (
         <View style={[globalStyles.alignItemsCenter, containerStyle]}>
-            <Avatar {...props} uri={avatar ?? uri} />
-            <Text style={globalStyles.mv10} category={TextCategory.H6}>
-                Upload a new photo
-            </Text>
-            <ImagePicker
-                onImageLoad={imageLoadHandler}
-                label="Choose photo"
-                containerStyle={buttonStyle}
-            />
+            {isSingleAvatarView ? (
```

Comment:
What do you mean by isSingleAvatarView

Reply:
Currently, AvatrPicker provides such a component.
Click on the button to download the image.
<img width="338" alt="Screenshot 2023-09-15 at 16 54 03" src="https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/70234990/310c0ddb-56d2-4686-8d94-53fa0cdc8705">
I wanted to extend it to the case when there is no button, only an image that you need to click to download.